digital_relay_server/__init__.py

```diff
@@ -69,6 +69,10 @@ def send_push_notifications(users, notification: PushNotification):
         if len(subscriptions) != len(user.push_subscriptions):
             user.push_subscriptions = subscriptions
             user.save()
+        if user.email_notifications:
+            with mail.connect() as connection:
+                connection.send(Message(subject="DXC RUN 4U Notifikácia: " + notification.title, body=notification.body,
+                                        recipients=[user.email]))
```

Comment:
Toto by asi mohol byť aj krajší ten mail, že sa tam rovno pošlú aj linky napríklad na stránku tímu a prípadne aj na to prevzatie štafety, napríklad cez takú nejakú šablónu ako v `send_email_invites()`